docs/Contributing/Understanding-host-vitals.md

```diff
@@ -480,7 +480,6 @@ SELECT
   version AS version,
   identifier AS extension_id,
   browser_type AS browser,
-  'Browser plugin (Chrome)' AS type,
```

Comment:
Was this removal intentional?

Reply:
Ah yes, wanted to make this for a long time (now is a good time given we'll QA the whole software feature), the `type` column is not used at all.